src/reistijden_v1/models.py

```diff
@@ -0,0 +1,77 @@
+from django.db import models
+
+
+class Publication(models.Model):
+    publication_type = models.CharField(max_length=255)  # one of travelTime or trafficFlow
```

Comment:
Is dit iets wat je eventueel wilt beperken (of in ieder geval verduidelijken) via `choices=...` ?

Reply:
Dat zou goed zijn, ware het niet dat ik verwacht dat er nog meer varianten komen die ik niet in de voorbeelden heb gekregen. Voor nu laat ik hem dus liever open. Maar ik zet er een TODO bij.